gammapy/spectrum/tests/test_simulation.py

```diff
@@ -71,3 +72,14 @@ def test_without_edisp(self):
         assert_allclose(np.sum(sim.npred_source.data.data.value),
                         167.467572145, rtol=0.01)
 
+    def test_without_aeff(self):
+        e_true = EnergyBounds.equal_log_spacing(1, 10, 5, u.TeV)
+        rate_model = self.source_model.copy()
+        rate_model.parameters['amplitude'].unit = u.Unit('TeV-1 s-1')
+        rate_model.parameters['amplitude'].value = 1
+        sim = SpectrumSimulation(source_model=rate_model,
+                                 livetime=4*u.h,
+                                 e_true=e_true 
+                                )
+        sim.simulate_obs(seed=23, obs_id=23)
+        assert sim.obs.on_vector.total_counts == 10509 * u.ct
```

Comment:
I thought we decided to *not* use `u.ct` or `u.ph` throughout Gammapy?
@joleroi - OK? Would you be willing to remove the `u.ct` here or in other places in `gammapy.spectrum` where it still exists?

Reply:
I can't remember that we had a discussion about this, let alone a decision. Can you point me somewhere?

Reply:
There is a discussion about pros and cons of having `u.ph` and / or `u.ct` or not here: #978
Do you dislike the solution to always omit them?
I think that is what we mostly have in Gammapy at the moment, and as I said in #978 I think it's the simplest solution.

Reply:
I missunderstood, I though you don't like the way the astropy units package is used. I'm in favour of always omitting ``u.ph`` or ``u.ct`` but I'm afraid ``gammapy.spectrum`` is full of them. I can take care of it.

Reply:
@joleroi - It's not so bad, I think it can be done in one pull request:
```
$ ack "u\.ct" gammapy --type=python
gammapy/spectrum/simulation.py
186:        self.on_vector.data.data += bkg_counts * u.ct

gammapy/spectrum/results.py
238:        data = self.npred_src * u.ct
248:            data = self.npred_bkg * u.ct

gammapy/spectrum/tests/test_core.py
16:        self.counts = [0, 0, 2, 5, 17, 3] * u.ct
26:        assert spec.data.data.unit.is_equivalent(u.ct)
32:        assert spec.data.data.unit.is_equivalent(u.ct)
66:        desired = [0, 7, 20] * u.ct
89:        assert spec.data.data.unit.is_equivalent(u.ct)
94:        assert spec.data.data.unit.is_equivalent(u.ct)

gammapy/spectrum/tests/test_simulation.py
44:        assert self.sim.obs.on_vector.total_counts == 160 * u.ct
51:        assert self.sim.obs.on_vector.total_counts == 530 * u.ct
52:        assert self.sim.obs.off_vector.total_counts == 1112 * u.ct
58:        assert self.sim.result[0].on_vector.total_counts == 158 * u.ct
59:        assert self.sim.result[1].on_vector.total_counts == 158 * u.ct
60:        assert self.sim.result[2].on_vector.total_counts == 161 * u.ct
61:        assert self.sim.result[3].on_vector.total_counts == 168 * u.ct
62:        assert self.sim.result[4].on_vector.total_counts == 186 * u.ct
70:        assert sim.obs.on_vector.total_counts == 161 * u.ct
85:        assert sim.obs.on_vector.total_counts == 10509 * u.ct

gammapy/spectrum/core.py
49:        data = np.arange(10) * u.ct
148:        self.data.data = binned_val * u.ct
267:        retval.data.data = counts_rebinned * u.ct
488:            data=counts_table['COUNTS'] * u.ct,
625:            kwargs['data'] = row['COUNTS'] * u.ct
$ ack "u\.ph" gammapy --type=python
<nothing>
```

Please add a changelog entry, as this likely is a change that will break a user script or two.